cake-factory/src/index.ts

```diff
@@ -0,0 +1,5 @@
+import { initServer } from './app.server'
+
+const port = parseInt(process.env.PORT, 10) || 3000;
```

Comment:
You can use the `+` sign to convert directly to number: `+(process.env.PORT || 3000)`